src/cclib/io/moldenwriter.py

```diff
@@ -16,22 +16,35 @@
 class MOLDEN(filewriter.Writer):
     """A writer for MOLDEN files."""
 
+    def __init__(self, ccdata, *args, **kwargs):
```

Comment:
Removed the init, made `required_attrs` a class constant, and moved the check to base class.